src/app/pages/CreateHabit/CreateHabit.module.css

```diff
@@ -0,0 +1,4 @@
+.createPage {
+  display: grid;
+  grid-template-rows: 1fr 1fr 1fr;
```

Comment:
Are you sure you need this?

Reply:
styling will be fixed in a dedicaded PR later on.